js/whitebit.js

```diff
@@ -1030,10 +1034,16 @@ module.exports = class whitebit extends Exchange {
                 if (isLimitOrder) {
                     // limit order
                     method = 'v4PrivatePostOrderNew';
+                    if (marginMode !== undefined) {
+                        method = 'v4PrivatePostOrderCollateralLimit';
```

Comment:
@Dan-krm should we throw if the user passes "isolated" as margin mode? Whitebit does not really use cross/isolated terms but we can assume there's only "cross" right?

Reply:
Yeah that makes sense, they seem to handle the margin trading like cross, any of the assets in the collateral account can be used as collateral for the different trading pairs.